modules/RTC/RTCUtils.js

```diff
@@ -119,6 +119,12 @@ function getConstraints(um, options) {
             constraints.video.optional.push({
                 sourceId: options.cameraDeviceId
             });
+        } else {
+            // by default prefer the "user" camera if we have not specified the
+            // exact device id for mobile devices
+            if (RTCBrowserType.isReactNative() || RTCBrowserType.isAndroid()) {
+                constraints.video.facingMode = "user";
```

Comment:
Why is the facingMode constraint applied on React Native and Android only? Don't we want a user-facing camera otherwise? Or maybe the constraint breaks Chrome or Firefox?